### PR TITLE
Assimilate my Akira-like blob!

### PR DESCRIPTION
**Pull request by Christian-QA**
Has a lot of cool stuff I definitely made myself #

### CODE REVIEW

File: ChrisIsAReallyCoolGuy.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+It's the truth
```
Comment (nickrstewarttds) [inline ChrisIsAReallyCoolGuy.txt:1]:
```suggestion
No.
```

File: src/com/qa/main/Animal.java [hunk 2]
```diff
@@ -4,8 +4,34 @@
 public class Animal {
 
 	// Fields
-	private int eyes = 12;
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (nickrstewarttds) [inline src/com/qa/main/Animal.java:8]:
```suggestion
//no
```

File: src/com/qa/main/Animal.java [hunk 3]
```diff
@@ -4,8 +4,34 @@
...
+	private int eyes = 2;
+=======
```
Comment (nickrstewarttds) [inline src/com/qa/main/Animal.java:10]:
```suggestion
//no
```

File: src/com/qa/main/Animal.java [hunk 4]
```diff
@@ -4,8 +4,34 @@
...
+	private int eyes = 15;
+>>>>>>> fourthsrc/master
+=======
```
Comment (nickrstewarttds) [inline src/com/qa/main/Animal.java:13]:
```suggestion
//no
```

Comment (nickrstewarttds) [inline src/com/qa/main/Animal.java:13]:
```suggestion
//no
```

File: src/com/qa/main/Animal.java [hunk 5]
```diff
@@ -4,8 +4,34 @@
...
+	private int eyes = 16;
+>>>>>>> sixthsrc/master
```
Comment (nickrstewarttds) [inline src/com/qa/main/Animal.java:15]:
```suggestion//no
```

File: ChrisIsAReallyCoolGuy.txt (hunk 1 above)
Comment (MDunnageQA) [inline ChrisIsAReallyCoolGuy.txt:1]:
```suggestion
WHY?
```

File: README.md [hunk 6]
```diff
@@ -3,6 +3,33 @@
 Hello my friends
 If you are seeing this, you've set up your fork correctly and fetched the changes.
 
+<<<<<<< HEAD
```
Comment (CarolineS-QA) [inline README.md]:
```suggestion
// stop forcing merges!
```

File: README.md [hunk 7]
```diff
@@ -3,6 +3,33 @@
...
+<<<<<<< HEAD
```
Comment (CarolineS-QA) [inline README.md]:
```suggestion
```suggestion
// You deserve the pain of editing one line at a time
```
```

File: README.md [hunk 8]
```diff
@@ -3,6 +3,33 @@
...
+>>>>>>> secondsrc/master
+=======
+Fun-with-merging
+Nick can't spell but Caroline can.
+
+>>>>>>> fifthsource/master
+=======
 Let's see if the .gitignore works
 
 ## second level heading
```
Comment (MDunnageQA) [inline README.md:35]: STOP YOU MADMAN

File: new2.txt [hunk 9]
```diff
@@ -0,0 +1 @@
+Nebulous Stuff
```
Comment (CarolineS-QA) [inline new2.txt:1]: This is okay I guess.

File: README.md [hunk 10]
```diff
@@ -20,3 +47,8 @@ public static void main(String[] args){
   System.out.println("Java is cool");
 }
 ```
+<<<<<<< HEAD
+>>>>>>> upstream/master
+=======
+pull request
+>>>>>>> secondsrc/master
```
Comment (MDunnageQA) [inline README.md:54]: BAD!

**Pull request merged**